docs/pages/cloud/architecture.mdx

```diff
@@ -47,6 +47,25 @@ The Teleport [proxy service](https://goteleport.com/docs/architecture/proxy/) ca
 
 The multi-region option is currently opt-in by default. Once you have an account, please reach out to your account manager, customer success engineer, or support@goteleport.com. A future update will expand the region availability and make all regions available by default.
 
+## Releases
+
+Teleport Cloud customers are typically running the latest release of the
+Teleport Auth and Proxy service.
+
+Teleport Cloud performs patch upgrades on a weekly basis and major upgrades on
```

Comment:
Teleport Cloud team upgrades the service with patch releases weekly and major releases quarterly. The team waits for the first minor release before a major upgrade. For example, the team will deploy (=teleport.major.version=).1.0 instead of (=teleport.major.version=).0.0. The first minor release happens 3-4 weeks after a first major release.